daprdocs/content/en/operations/security/app-api-token.md

```diff
@@ -89,14 +89,14 @@ kubectl rollout restart deployment/<deployment-name> --namespace <namespace-name
 
 ## Authenticating requests from Dapr
 
-Once app token authentication is configured in Dapr, all requests *coming from Dapr* include the token.
+Once app token authentication is configured using the environment variable or Kubernetes secret `app-api-token`, the Dapr sidecar always includes the HTTP header/gRPC metadata `dapr-api-token: <token>` in the calls to the app. From the app side, ensure you are authenticating using the `dapr-api-token` value, even though you set `app-api-token` to enable API authentication.
```

Comment:
```suggestion
Once app token authentication is configured using the environment variable or Kubernetes secret `app-api-token`, the Dapr sidecar always includes the HTTP header/gRPC metadata `dapr-api-token: <token>` in the calls to the app. From the app side, ensure you are authenticating using the `dapr-api-token` value which uses the `app-api-token` you set to authenticate requests from Dapr.
```

Reply:
Given we are update these sections I would also update https://docs.dapr.io/operations/security/api-token/#adding-api-token-to-client-api-invocations to say 

"Once token authentication is configured in Dapr, all clients invoking Dapr API need to append the `dapr-api-token` token token to every request. Note that the Dapr SDKs read [DAPR_API_TOKEN](https://docs.dapr.io/reference/environment/) and set this by default for you.

Also to help this I would add a diagram. It is an easy diagram that shows the Dapr sidecar reading the DAPR_API_TOKEN (or annotation) and/or the APP_API_TOKEN (or annotation). These set up the authentication tokens.

Then on the request from the App-> Dapr, Dapr checks the `dapr-api-token` matches to be the same as DAPR_API_TOKEN or from Dapr->App, the App checks that `dapr-api-token` matches to be the same as APP_API_TOKEN. Could you draw this picture?